packages/client/test/lib.spec.js

```diff
@@ -137,6 +137,7 @@ describe('client', () => {
     })
 
     it('upload large CAR with a CarReader', async function () {
+      // @ts-ignore
```

Comment:
This blows up the linter so we had to add it here to get the build to pass. It's in a spec file, so there are no negative downstream side effects we need to worry about.